.travis.yml

```diff
@@ -77,5 +77,8 @@ matrix:
       env: SCALIKEJDBC_DATABASE="mysql"
       jdk: openjdk11
    # Scala 2.13.0 Milestone Releases
-    - scala: 2.13.0-RC1
-      env: SCALIKEJDBC_DATABASE="mysql"
+    - scala: 2.13.0-RC2
+      script:
+      - sbt
+        "++${TRAVIS_SCALA_VERSION}!"
+        "all config/test core/test interpolation/test interpolation-macro/test joda-time/test library/test mapper-generator-core/test syntax-support-macro/test"
```

Comment:
- This pull request use scalatest 3.1.x if Scala 2.13.x
- scalatest 3.1.x separate other library dependencies (e.g. scalacheck, mockito, testng)
- scalikejdbc-stream module use scalatestplus-testng, but not released yet for Scala 2.13.0-RC2
  - https://repo1.maven.org/maven2/org/scalatestplus/scalatestplus-testng_2.13.0-RC2/
  - https://repo1.maven.org/maven2/org/scalatestplus/scalatestplus-testng_2.13.0-RC1/
  - https://github.com/scalatest/scalatestplus-testng

Reply:
Fair enough. Thanks.